Doc/library/typing.rst

```diff
@@ -158,9 +158,6 @@ Type aliases are useful for simplifying complex type signatures. For example::
            servers: Sequence[tuple[tuple[str, int], dict[str, str]]]) -> None:
        ...
 
-Note that ``None`` as a type hint is a special case and is replaced by
-``type(None)``.
-
```

Comment:
This just doesn't fit in this section at all. Even if it did, it's incorrect: `None` is treated more like `Literal[None]` than `NoneType` in terms of typing